omaha/net/cup_ecdsa_request_unittest.cc

```diff
@@ -34,14 +34,14 @@
 namespace omaha {
 
 const TCHAR* const kGetUrl =
-    _T("http://tools.google.com/service/update2/id");
+    _T("https://tools.") COMPANY_DOMAIN _T("/service/update2/id");
 const TCHAR* const kGetUrlNoResponseBody =
-    _T("http://tools.google.com/service/update2/nil");
+    _T("https://tools.") COMPANY_DOMAIN _T("/service/update2/nil");
 const TCHAR* const kPostUrl =
-    _T("http://tools.google.com/service/update2");
+    _T("https://tools.") COMPANY_DOMAIN _T("/service/update2");
 const TCHAR* const kPostHttpsUrl =
-    _T("https://tools.google.com/service/update2");
-const uint8 kRequestBuffer[] = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><o:gupdate xmlns:o=\"http://www.google.com/update2/request\" protocol=\"2.0\" version=\"1.2.1.0\" ismachine=\"1\" testsource=\"dev\"><o:os platform=\"win\" version=\"5.1\" sp=\"Service Pack 2\"/><o:app appid=\"{52820187-5605-4C18-AA51-8BD0A1209C8C}\" version=\"1.1.1.3\" lang=\"abc\" client=\"{0AF52D61-9958-4fea-9B29-CDD9DCDBB145}\" iid=\"{F723495F-8ACF-4746-8240-643741C797B5}\"><o:event eventtype=\"1\" eventresult=\"1\" errorcode=\"0\" previousversion=\"1.0.0.0\"/></o:app></o:gupdate>";  // NOLINT
+    _T("https://updates.") COMPANY_DOMAIN _T("/service/update2");
+const uint8 kRequestBuffer[] = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><o:gupdate xmlns:o=\"http://www.") COMPANY_DOMAIN _T("/update2/request\" protocol=\"2.0\" version=\"1.2.1.0\" ismachine=\"1\" testsource=\"dev\"><o:os platform=\"win\" version=\"5.1\" sp=\"Service Pack 2\"/><o:app appid=\"{52820187-5605-4C18-AA51-8BD0A1209C8C}\" version=\"1.1.1.3\" lang=\"abc\" client=\"{0AF52D61-9958-4fea-9B29-CDD9DCDBB145}\" iid=\"{F723495F-8ACF-4746-8240-643741C797B5}\"><o:event eventtype=\"1\" eventresult=\"1\" errorcode=\"0\" previousversion=\"1.0.0.0\"/></o:app></o:gupdate>";  // NOLINT
```

Comment:
There is a compile error here:
```
net\cup_ecdsa_request_unittest.cc(44): error C2059: syntax error: ')'
net\cup_ecdsa_request_unittest.cc(44): fatal error C1057: unexpected end of file in macro expansion
```

Reply:
Something like this compiles `"...." COMPANY_DOMAIN_ANSI "..."` but maybe not brand the XML namespace to begin with, that would be my suggestion.

Reply:
With this PR, the unit tests fail:

```
[----------] Global test environment tear-down
[==========] 3527 tests from 236 test suites ran. (294144 ms total)
[  PASSED  ] 3525 tests.
[  FAILED  ] 2 tests, listed below:
[  FAILED  ] OmahaCustomizationTest.Constants_BuildFiles
[  FAILED  ] CupEcdsaRequestTest.PostSimpleRequestHttps
```

Reply:
Oops... that was a mistake on my part. Fixing...

Reply:
I've fixed the compilation error (and restored the xml namespace to google.com) and the Constants_BuildFiles test failure. I'm not sure about the PostSimpleRequestHttps failure, or how that's related.

Reply:
I am guessing right now (I will investigate in a bit) but one oddity about Omaha unit tests is that there are some tests which actually use the network stack and connect to actual servers instead of using network mocks. It's possible that branding has resulted in changing the url of a Google server with something else, so the test fails because it can't connect.

Reply:
If my branding factoring was done correctly, then there should be no functional difference when built by Google. So the question is: did I make any additional errors anywhere else that might cause this?

Reply:
The error is 12007, which is name not resolved. I suggest not changing the urls in the unit test code for ECDSA CUP. On the positive side, it talks to a Google backend which provides a de-facto reference implementation for the
signing protocol.